### PR TITLE
Improve the JavaDoc in io.quarkus.arc.processor.QualifierRegistrar

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

File: independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/QualifierRegistrar.java [hunk 1]
```diff
@@ -12,7 +12,8 @@
 
     /**
      * Returns a map of additional qualifers where the key represents the annotation type and the value is an optional set of
-     * non-binding members.
+     * non-binding members. Here, "non-binding" is meant in the sense of `javax.enterprise.util.Nonbinding`. I.e. members named
```
Comment (mkouba) [inline independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/QualifierRegistrar.java:15]: This is javadoc, we should probably use something like `{@link javax.enterprise.util.Nonbinding}` or `{@code javax.enterprise.util.Nonbinding}` instead...

Comment (ppalaga) [reply independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/QualifierRegistrar.java:15]: +1, sorry for selecting a bad serializer :)

Comment (ppalaga) [reply independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/QualifierRegistrar.java]: Done in dd58052

**Comment by ppalaga** [comment]
dd58052: addressed the comment of @mkouba

**Review by mkouba — APPROVED** [review]
Looks good.

**Pull request merged**